changelogs/fragments/vaultfilters.yml

```diff
@@ -0,0 +1,5 @@
+add plugin.filter:
```

Comment:
This should be in a `minor_changes` section and formatted for rst

Reply:
This is a Newish Thing that got added fairly recently: https://docs.ansible.com/ansible/devel/community/development_process.html#changelog-fragment-entry-format-for-new-jinja2-plugins-roles-and-playbooks

Reply:
It will make the filter show up under "New Plugins" like other new plugins: https://github.com/ansible-collections/community.general/blob/stable-3/CHANGELOG.rst#new-plugins-1

Reply:
Oh, nice!